src/toolkits/activity_classification/ac_data_iterator.hpp

```diff
@@ -179,6 +182,9 @@ class simple_data_iterator: public data_iterator {
   gl_sframe_range range_iterator_;
   gl_sframe_range::iterator next_row_;
   gl_sframe_range::iterator end_of_rows_;
+  size_t sample_in_row_;
+  bool is_train_;
+  bool use_data_augmentation_;
```

Comment:
Let’s initialize these too. Unless another value makes sense, you can use `false` or 0 as the default initializer.